Samples/ProvisionDemoLab/azuredeploy.parameters.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newLabName": {
+      "value": "FabrikamDemoLab"
+    },
+    
+    // parameters to set lab policies

```

Comment:
I've ran the ARM template without problems while having comments in the JSON. However, I've removed the comments just the same.